packages/components/src/components/modal/modal.jsx

```diff
@@ -34,27 +40,21 @@ class ModalElement extends React.PureComponent {
 
     render() {
         return ReactDOM.createPortal(
-            <div ref={this.setWrapperRef} id={this.props.id} className={classNames('modal__container', this.props.className && `modal__${this.props.className}`)}>
-                <div className='modal-header'>
-                    <h3 className='modal-header__sidebar'>{this.props.title}</h3>
-                    <div className='modal-header__main'>
-                        { this.props.header &&
-                            <div className='modal-header__section'>
-                                {this.props.header}
-                            </div>
-                        }
-                        <div id='dt_modal_close_icon' className='modal-header__close' onClick={this.props.toggleModal}>
-                            <Icon icon='ModalIconClose' />
-                        </div>
+            <div ref={this.setWrapperRef} id={this.props.id} className={classNames('modal__container', this.props.className && `modal__container_${this.props.className}`)}>
+                <div className={classNames('modal-header', this.props.className && `modal-header__${this.props.className}`)}>
```

Comment:
Would you please use the `dc-[component]` prefix?

Reply:
@reza-fs sure. i forgot it again :)